strongbox-security/strongbox-security-api/src/main/java/org/carlspring/strongbox/security/UserPathPermissions.java

```diff
@@ -21,6 +24,7 @@
 {
 
     @XmlElement(name = "path")
+    @OneToMany(cascade = CascadeType.ALL)
```

Comment:
as above, probably unnecessary+ `UserPathPermissions` should not be a `GenericEntity`, unless I've missed something